### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pedro-101**
Replaced non coding references to ConnectWise Automate with Labtech.

### CODE REVIEW

**Review by chrisdeke — CHANGES_REQUESTED** [review]
42: This is incorrect, it should either be Connectwise Automate or Labtech not just "Connectwise"
74: This will break the code example?

File: README.md [hunk 1]
```diff
@@ -28,18 +28,18 @@ Known Issues:
...
+- **ScreenConnect (ConnectWise Control) API Support** - Planned :chart_with_upwards_trend:
 - **Batch requests** - Planned :chart_with_upwards_trend:
 
 How-to:
 =============
 - [Install](#install)
 - [Initializing the API Clients](#initialize-api-client)
   - [ConnectWise Manage](#connectwise-manage)
-  - [ConnectWise Automate](#connectwise-automate)
+  - [ConnectWise ](#connectwise-)
```
Comment (Yoshify) [inline README.md:42]: This breaks the hyper link to line 72

File: README.md [hunk 2]
```diff
@@ -69,22 +69,22 @@ manage_api_client = ConnectWiseManageAPIClient(
...
-### ConnectWise Automate
+### Labtech
 ```python
-from pyconnectwise import ConnectWiseAutomateAPIClient
+from pyconnectwise import ConnectWiseAPIClient
 
 # init client
-automate_api_client = ConnectWiseAutomateAPIClient(
-  # your automate url
+_api_client = ConnectWiseAPIClient(
```
Comment (Yoshify) [inline README.md:77]: Lets keep the code example as is with automate_api_client

File: README.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
 [![Health IT Logo](https://healthit.com.au/wp-content/uploads/2019/06/HIT-proper-logo.png)](https://healthit.com.au)
 
-# pyConnectWise - An API library for ConnectWise Manage and ConnectWise Automate, written in Python
+# pyConnectWise - An API library for ConnectWise Manage and Labtech (ConnectWise Automate), written in Python
```
Comment (Yoshify) [inline README.md:3]: I'm more inclined to reversing this and having it be ConnectWise Automate (Labtech) as having the current name (former name) makes more sense.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
-pyConnectWise is a full-featured, type annotated API client written in Python for the ConnectWise APIs based off their OpenAPI schemas. 
+pyConnectWise is a full-featured, type annotated API client written in Python for the ConnectWise APIs based on their OpenAPI schemas. 
 
 This library has been developed with the intention of making the ConnectWise APIs simple and accessible to non-coders while allowing experienced coders to utilize all features the API has to offer without the boilerplate.
 
-pyConnectWise currently supports both ConnectWise Manage and ConnectWise Automate.
+pyConnectWise currently supports both ConnectWise Manage and Labtech (ConnectWise Automate).
```
Comment (Yoshify) [inline README.md:9]: See review for line 3.

**Review by Yoshify — CHANGES_REQUESTED** [review]
Couple changes to make, then we can merge.

**Pull request closed without merging**